gstreamer-rtsp-server/src/auto/r_t_s_p_stream_transport.rs

```diff
@@ -0,0 +1,143 @@
+// This file was generated by gir (37f7805+) from gir-files (???)
+// DO NOT EDIT
+
+use RTSPStream;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use gst;
+use gst_rtsp;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct RTSPStreamTransport(Object<ffi::GstRTSPStreamTransport, ffi::GstRTSPStreamTransportClass>);
+
+    match fn {
+        get_type => || ffi::gst_rtsp_stream_transport_get_type(),
+    }
+}
+
+impl RTSPStreamTransport {
+    //pub fn new(stream: &RTSPStream, tr: /*Ignored*/&mut gst_rtsp::RTSPTransport) -> RTSPStreamTransport {
+    //    unsafe { TODO: call ffi::gst_rtsp_stream_transport_new() }
+    //}
+}
+
+pub trait RTSPStreamTransportExt {
+    fn get_rtpinfo(&self, start_time: gst::ClockTime) -> Option<String>;
+
+    fn get_stream(&self) -> Option<RTSPStream>;
+
+    //fn get_transport(&self) -> /*Ignored*/Option<gst_rtsp::RTSPTransport>;
```

Comment:
This seems useful to add to libgstrtsp (GstRTSPTransport)

Reply:
yup, will do in later PRs